Frameworks/CoreText/CTFont.mm

```diff
@@ -692,13 +704,50 @@ CGGlyph CTFontGetGlyphWithName(CTFontRef font, CFStringRef glyphName) {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes  kCTFontVerticalOrientation is unsupported, returns the same as default and horizontal orientations
 */
 CGRect CTFontGetBoundingRectsForGlyphs(
     CTFontRef font, CTFontOrientation orientation, const CGGlyph glyphs[], CGRect* boundingRects, CFIndex count) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (!glyphs || count <= 0) {
+        TraceError(g_logTag, L"No glyphs were specified");
+        return CGRectNull;
+    }
```

Comment:
?

---
In reply to: [83340871](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r83340871) [](ancestors = 83340871)